src/core-taggeds-optional/Optional.Tests/TestData/IAction.cs

```diff
@@ -0,0 +1,33 @@
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
There are at least a couple of duplicates `[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]` in different files.
Shall we consider creating a single file kind of `Assembly.cs` with `[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]`  in the test project root?

Reply:
Created the separate file Assembly.cs for this purpose